tests/utils/PolygonUtilsTest.cpp

```diff
@@ -145,12 +155,13 @@ TEST_F(MoveInsideTest, cornerEdgeTest2)
 
     constexpr coord_t maximum_error = 10;
     ASSERT_TRUE(vSize(result - supposed1) <= maximum_error || vSize(result - supposed2) <= maximum_error)
-        << close_to << " moved with " << distance << " micron inside to " << result << " rather than " << supposed1 << " or " << supposed2 << ".";
+      << close_to << " moved with " << distance << " micron inside to " << result << " rather than " << supposed1 << " or " << supposed2
+      << ".";
```

Comment:
Indent with 4 spaces.

I also think putting the period on a separate line makes no sense for readability.